nixos/modules/services/web-apps/ethercalc.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ethercalc;
+in {
+  options = {
+    services.ethercalc = {
+      enable = mkEnableOption "ethercalc, an online collaborative spreadsheet server (persistent state will be maintained under <filename>/var/lib/ethercalc</filename>)";
```

Comment:
not sure the comment on where state lives is necessary, `/var/lib/${serviceName}` is quite a common pattern.

Reply:
This comment serves a further purpose: ethercalc can also use a redis serve for storing its state. By mentioning this directory, we're making it clear that this is just a simple, first usable version of the service description, where we fall back to the plain files store.

Reply:
ok, that makes sense.